test/connectors/tinyman/tinyman.test.ts

```diff
@@ -0,0 +1,258 @@
+      },
+      validatorAppID: 1002541853,
+      asset1ID: 31566704,
+      asset2ID: 0,
+      status: 'ready',
+      contractVersion: 'v2',
+      asset1ProtocolFees: BigInt('10961420982'),
+      asset2ProtocolFees: BigInt('48196942141'),
+      asset1Reserves: BigInt('1200097723737'),
+      asset2Reserves: BigInt('6633822022882'),
+      issuedPoolTokens: BigInt('2690871970933'),
+      cumulativePriceUpdateTimeStamp: 1682547803,
+      protocolFeeRatio: 6,
+      totalFeeShare: BigInt('30'),
+      poolTokenID: 1002590888,
+    };
+  });
+};
+
+describe('verify Tinyman estimate Sell Trade', () => {
+  it('Should return an ExpectedTrade when available', async () => {
+    patchFetcher();
+
+    const expectedTrade = await tm.estimateTrade({
+      chain: CHAIN_NAME,
+      network: NETWORK,
+      base: 'ALGO',
+      quote: 'USDC',
+      amount: '1',
+      side: 'SELL',
+    });
+    expect(expectedTrade).toHaveProperty('trade');
+    expect(expectedTrade).toHaveProperty('expectedAmount');
+  });
+
+  it('Should throw an error if no pair is available', async () => {
+    patchFetcher();
+
+    await expect(async () => {
+      await tm.estimateTrade({
+        chain: CHAIN_NAME,
+        network: NETWORK,
+        base: 'ETH',
+        quote: 'DAI',
+        amount: '1',
+        side: 'SELL',
+      });
+    }).rejects.toThrow('');
+  });
+});
+
+describe('verify Tinyman estimate Buy Trade', () => {
+  it('Should return an ExpectedTrade when available', async () => {
+    patchFetcher();
+
+    const expectedTrade = await tm.estimateTrade({
+      chain: CHAIN_NAME,
+      network: NETWORK,
+      base: 'ALGO',
+      quote: 'USDC',
+      amount: '1',
+      side: 'BUY',
+    });
+    expect(expectedTrade).toHaveProperty('trade');
+    expect(expectedTrade).toHaveProperty('expectedAmount');
+  });
+
+  it('Should return an error if no pair is available', async () => {
+    patchFetcher();
+
+    await expect(async () => {
+      await tm.estimateTrade({
+        chain: CHAIN_NAME,
+        network: NETWORK,
+        base: 'ETH',
+        quote: 'DAI',
+        amount: '1',
+        side: 'BUY',
+      });
+    }).rejects.toThrow('');
+  });
+});
+
+describe('getAllowedSlippage', () => {
+  it('return value from config when string is null', () => {
+    const allowedSlippage = tm.getSlippage();
+    expect(allowedSlippage).toEqual(0.02);
+  });
+});
+
+describe('verify Tinyman executeTrade', () => {
+  it('Should pass when pair is available', async () => {
+    patchFetcher();
+    patchSendHelper();
+
+    const trade = await tm.estimateTrade({
+      chain: CHAIN_NAME,
+      network: NETWORK,
+      base: 'ALGO',
+      quote: 'USDC',
+      amount: '1',
+      side: 'BUY',
+    });
+
+    const tradeResult = await tm.executeTrade(
+      algorand.getAccountFromPrivateKey(MNEUMONIC),
+      trade.trade,
+      true
+    );
+    expect(tradeResult.txnID).toEqual(123);
+    expect(tradeResult.round).toEqual(1);
+  });
+});
```

Comment:
I've ran `yarn install && yarn build`, but this test fails on my end with
```
l.account.address is not a function
TypeError: l.account.address is not a function
    at /Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:69807
    at l (/Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:1348)
    at Generator.<anonymous> (/Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:2689)
    at Generator.next (/Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:1711)
    at s (/Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:7342)
    at i (/Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:7545)
    at /Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:7604
    at new Promise (<anonymous>)
    at Object.<anonymous> (/Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:7485)
    at Object.or (/Users/petioptrv/development/gateway/node_modules/@tinymanorg/tinyman-js-sdk/dist/index.js:1:70929)
```

Reply:
done